.travis.yml

```diff
@@ -12,6 +12,13 @@ env:
 jobs:
   include:
   - name: Test Go stable
+  - name: Test Docker plugin
+    before_script:
+    - go install
+    - docker pull busybox
+    - docker-compose -f examples/swarm/docker-compose.yml up -d
+    script:
+    - wash validate docker
```

Comment:
I have seen a transient with this. Will dig into it a bit.

Reply:
Ok, if it hits a symlink (like `/var/run` in the web server from this example) or a write-only file (like `/proc/sysrq-trigger`) it treats it as a file and tries to read/stream from it. So need to
- [ ] update write-only files to not support read/stream
- [ ] fix whatever's wrong with symlinks, we're likely ignoring part of the stat results or need to pass an argument to stat

Reply:
I think I'm going to pull out the .travis.yml changes and include them when I fix #376.